usr/local/share/deskpro/container-var-reference.json

```diff
@@ -1,419 +1,711 @@
+    "description": "When DESKPRO_SITE_INFO_SOURCE=httpheader, this is the header that contains the site info.",
     "type": "string",
     "setEnv": true
   },
   {
-    "description": "One of: httpheader, file, env",
     "name": "DESKPRO_SITE_INFO_SOURCE",
+    "description": "One of: httpheader, file, env",
     "type": "string",
+    "example": "httpheader, file, env",
     "setEnv": true
   },
   {
-    "description": "",
-    "name": "DESKPRO_STORAGE_SETTINGS"
+    "name": "DESKPRO_STORAGE_SETTINGS",
+    "description": "A JSON-encoded string of storage adapter settings according to the type.",
+    "type": "string"
   },
   {
-    "description": "",
-    "name": "DESKPRO_STORAGE_TYPE"
+    "name": "DESKPRO_STORAGE_TYPE",
+    "description": "Storage type:",
+    "type": "string",
+    "example": "db, fs, or s3",
+    "default": "db"
   },
   {
+    "name": "DESKPRO_SVC_KEY",
     "description": "Similar to DESKPRO_APP_KEY but can be used for multi-tenant services that need secrets before a tenant context is available.",
-    "name": "DESKPRO_SVC_KEY"
+    "type": "string"
   },
   {
     "name": "DESKPRO_TENANT_ID",
-    "type": "string",
-    "description": "An ID given to this instance of Deskpro. (Single-tenant value)"
+    "description": "An ID given to this instance of Deskpro. (Single-tenant value)",
+    "type": "string"
   },
   {
-    "description": "",
-    "name": "DESKPRO_TEST_RESET_DB_SCRIPT"
+    "name": "DESKPRO_TEST_RESET_DB_SCRIPT",
+    "description": "Path to an executable to run on the DB reset endpoint.",
+    "type": "string"
   },
   {
+    "name": "DESKPRO_TWITTER_APP_ID",
     "description": "",
-    "name": "DESKPRO_TWITTER_APP_ID"
+    "type": "string"
   },
   {
+    "name": "DESKPRO_TWITTER_APP_KEY",
     "description": "",
-    "name": "DESKPRO_TWITTER_APP_KEY"
+    "type": "string"
   },
   {
+    "name": "DESKPRO_TWITTER_APP_KEY_SECRET",
     "description": "",
-    "name": "DESKPRO_TWITTER_APP_KEY_SECRET"
+    "type": "string"
   },
   {
     "name": "HEALTHCHECK_TEST_DB_CONNECTION",
-    "type": "boolean",
-    "description": "Enable checking the database connection with the healthcheck. This confirms that the container can connect to the configured database."
+    "description": "Enable checking the database connection with the healthcheck. This confirms that the container can connect to the configured database.",
+    "type": "boolean"
   },
   {
     "name": "HEALTHCHECK_TEST_DISCOVER",
-    "type": "boolean",
-    "description": "Enable checking the discover endpoint with the healthcheck. This confirms that the container can connect to the database and serve a web request which can be a good indicator of health."
+    "description": "Enable checking the discover endpoint with the healthcheck. This confirms that the container can connect to the database and serve a web request which can be a good indicator of health.",
+    "type": "boolean"
   },
   {
     "name": "HTTP_INTERNAL_MODE",
```

Comment:
I know this already existed in the file, but it's also internal so I think we should remove